code-model-comparison/code_ingestion.py

```
@@ -0,0 +1,18 @@
+from gitingest import ingest
+from typing import Dict, Any
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)